code/cheats_table/cheats_table.cpp

```diff
@@ -0,0 +1,110 @@
+#include "cheats_table.h"
+#include "parse/parselo.h"
+
+void cheat_table_init() {
+	customCheats.clear();
+
+	// Add default Spawn Ship Cheat (Volition Bravos).
+	// Won't interfere with Total Conversions, since if the ship doesn't exist, the game will just ignore the cheat.
+	// The other default cheats do not use the Custom Cheat Framework and are still handled in keycontrol.cpp
+	std::unique_ptr<CustomCheat> bravos(new SpawnShipCheat("arrrrwalktheplank", "Walk the plank", false, "Volition Bravos", "Volition Bravos"));
+	customCheats.emplace(SCP_string(bravos->cheatCode), std::move(bravos));
```

Comment:
instead of hard-defining the cheat like this here, could I suggest a similar method as with default game_settings.tbl / fonts.tbl / etc,, located in code/def_files/data/tables.
This would allow any mod to override otherwise built-in cheats or just outright remove them altogether.